lib/cylc/graph_parser.py

```diff
@@ -674,11 +672,15 @@ def test_bad_node_syntax(self):
         The correct format is:
           NAME(<PARAMS>)([CYCLE-POINT-OFFSET])(:TRIGGER-TYPE)")
         """
-        gp = GraphParser()
+        params = {'m': ['0', '1'], 'n': ['0', '1']}
+        templates = {'m': '_m%(m)s', 'n': '_n%(n)s'}
+        gp = GraphParser(parameters=(params, templates))
         self.assertRaises(
             GraphParseError, gp.parse_graph, "foo[-P1Y]<m,n> => bar")
         self.assertRaises(
             GraphParseError, gp.parse_graph, "foo[-P1Y]<m,n> => bar")
```

Comment:
These two are identical - but not your fault and it doesn't really matter; we can fix it in a subsequent PR.